src/components/Comment/Comment.tsx

```diff
@@ -507,24 +499,30 @@ export const Comment = ({
                           onClick={() => setCommentBeingRepliedTo(comment)}
                           className={cx(
                             commentControlsButtonStyles,
-                            removePaddingLeft
+                            removePaddingLeft,
+                            flexRowStyles
```

Comment:
We wouldn't need these if we use the Link component from source foundation?

Reply:
this is a button HTML element, so I am unsure what you mean. Nonetheless it does raise the point that we should convert the buttons here to src foundations. I prefer to do it in another PR